src/components/UpdateProfile/UpdateProfileStyled.ts

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+import * as colors from 'styles/variables';
+
+export const WrapStyled = styled.div.attrs(() => ({
+  className: 'msd__user-profile__wrapper'
+}))`
+  * {
+    box-sizing: border-box;
+  }
+  box-sizing: border-box;
+  width: 100%;
+  padding: 20px;
+  background: ${(props) =>
+    props.theme.backgroundColor || colors.BackgroundColor};
+`;
+
+export const SectionStyled = styled.section``;
```

Comment:
is this needed? seems like a good candidate for removal, since it doesn't define any styles

Reply:
It depends on if we want to have fancy looking `<SectionStyled>` or just boring `<section>` 😛

Reply:
Will remove it